tagged-localbuild.xml

```diff
@@ -0,0 +1,764 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="NXP" fetch="https://github.com/NXP/"/>
+  <remote name="aosp" review="https://android-review.googlesource.com/" fetch="https://android.googlesource.com/"/>
+  <remote name="exfatprogs" fetch="https://github.com/exfatprogs/"/>
+  <remote name="hybris" fetch="https://github.com/mer-hybris"/>
+  <remote name="sony" fetch="https://github.com/sonyxperiadev/"/>
+
+  <default remote="aosp" revision="refs/tags/android-11.0.0_r46" sync-j="4"/>
+
+  <project name="QcRilAm" path="vendor/oss/qcrilam" remote="sony" revision="ef51ec65f5609e0fe9180ec3d2d054b72c0b8bee" upstream="master" dest-branch="master" groups="device,local::oss"/>
```

Comment:
Hmm, I could not find local::* groups neither in Sony's nor AOSP's manifests
```suggestion
  <project name="QcRilAm" path="vendor/oss/qcrilam" remote="sony" revision="ef51ec65f5609e0fe9180ec3d2d054b72c0b8bee" upstream="master" dest-branch="master" groups="device"/>
```

Reply:
> That's normal because Sony repos come from local manifest.
See here:
https://gerrit.googlesource.com/git-repo/+/HEAD/docs/manifest-format.md
https://gerrit.googlesource.com/git-repo/+/78f4dd3138b774dfea9d789d7324b8857f5a1a58

But we do not use local_manifests any longer (only during the very first creation of an adaptation manifest), thus diffing with them will be hard.

Reply:
> 
Repo inserts the tag for repos that come from the local manifest, unless you want to edit the manifest every time we update the manifest from Sony's changes by we should just keep them.

Reply:
It depends which manifest update method is used more often: resync from scratch from Sony (requires repo init/sync), or manually diff parts of manifest (can be done just via github browser).

Reply:
It depends which manifest update method is used more often: resync from scratch from Sony (requires repo init/sync), or manually diff parts of manifest (can be done just via github browser).